src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs

```diff
@@ -177,7 +177,7 @@ public void reader_14()
             ReadFrom(xml);
         }
 
-        //XElement: non-wellformed xml
+        //XElement: non�well-formed xml
```

Comment:
We shouldn't change force an encoding of the file just to use the "right" dash in a comment.  In general we're trying to use only ASCII characters in the code files.